Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <PropertyGroup>
+    <!-- For non-SDK projects that import this file and then import Microsoft.Common.props,
+         tell Microsoft.Common.props not to import Directory.Build.props again. -->
+    <ImportDirectoryBuildProps>false</ImportDirectoryBuildProps>
+    <!-- Enable .NET code analysis (this is necessary for projects that target 3.1 or earlier). -->
+    <EnableNETAnalyzers>true</EnableNETAnalyzers>
```

Comment:
Consider AnalysisLevel latest and AnalysisMode AllEnabledByDefault in a follow-up PR in case the number of violations isn't big.

Reply:
There are too many warnings with AllEnabledByDefault.